keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -661,47 +662,35 @@ The following audit annotations will be added:
 5. `pod-security.kubernetes.io/exempt = [user, namespace, runtimeClass]` For exempt requests, record the parameters
    that triggered the exemption here.
 
-### PodSecurityPolicy Migration
-
-<<[UNRESOLVED]>>
 
-_Targeting Beta or GA, non-blocking for Alpha._
+### PodSecurityPolicy Migration
```

Comment:
updated with the documentation added in 1.22, moved automation to potential future work